machine_learning_hep/validation/validation.py

```diff
@@ -0,0 +1,63 @@
+#############################################################################
+##  © Copyright CERN 2018. All rights not expressly granted are reserved.  ##
+##                 Author: Gian.Michele.Innocenti@cern.ch                  ##
+## This program is free software: you can redistribute it and/or modify it ##
+##  under the terms of the GNU General Public License as published by the  ##
+## Free Software Foundation, either version 3 of the License, or (at your  ##
+## option) any later version. This program is distributed in the hope that ##
+##  it will be useful, but WITHOUT ANY WARRANTY; without even the implied  ##
+##     warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.    ##
+##           See the GNU General Public License for more details.          ##
+##    You should have received a copy of the GNU General Public License    ##
+##   along with this program. if not, see <https://www.gnu.org/licenses/>. ##
+#############################################################################
+
+"""
+Script base function for validation histograms
+"""
+
+# pylint: disable=too-many-lines
+# pylint: disable=import-error, no-name-in-module
+from utilities_plot import makefill1dhist, makefill2dhist
+
+
+class ValidationCollection:
+    """
+    Class for managing histograms of validation
+    """
+
+    def __init__(self, dataframe=None, tag="", verbose=False):
+        self.source_dataframe = dataframe
+        self.collection_tag = tag
+        self.histograms = []
+        self.verbose = verbose
+
+    def reset(self, dataframe, tag):
```

Comment:
In `__init__` you have default arguments. Why not here?

Reply:
if one resets the validation he has to provide a tag, otherwise the risk if of doubling the histograms name

Reply:
Ok, makes sense. But maybe then giving it another name? `reset` might be misleading (sorry I know I am super picky)

Reply:
Agreed